code/lib/instrumenter/src/instrumenter.ts

```diff
@@ -112,6 +114,10 @@ export class Instrumenter {
       isPlaying?: boolean;
       isDebugging?: boolean;
     }) => {
+      if (!storyId) {
+        return;
+      }
+
       const state = this.getState(storyId);
       this.setState(storyId, {
```

Comment:
I think this might break the code. It seems like getState and setState are able to handle a nullable storyId, so I think it is more safe to widen the the type of those methods instead.